From f4152dbacbebcdbf55c9ffb7fccba3b0cfe9af48 Mon Sep 17 00:00:00 2001
From: Daniel Nachun <daniel.nachun@gmail.com>
Date: Sun, 29 Jun 2025 15:32:22 -0700
Subject: [PATCH] remove unneeded LDFLAGS

---
 src/Makevars.ucrt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars.ucrt b/src/Makevars.ucrt
index f50cde32..18fe6fbd 100644
--- a/src/Makevars.ucrt
+++ b/src/Makevars.ucrt
@@ -6,7 +6,7 @@ PKG_CPPFLAGS = \
 ifeq (,$(shell pkg-config --version 2>/dev/null))
   LIBBROTLI = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libbrotlidec.a),-lbrotlidec -lbrotlicommon),)
   PKG_LIBS += \
-	-lfreetype -lharfbuzz -lfreetype $(LIBBROTLI) -lpng -lbz2 -lz 
+	-lfreetype -lfreetype $(LIBBROTLI) -lpng -lbz2
   PKG_CPPFLAGS += -I$(R_TOOLS_SOFT)/include/freetype2
 else
   PKG_LIBS += $(shell pkg-config --libs freetype2)
-- 
2.49.0

